c/kastore.h

```diff
@@ -94,8 +94,9 @@ Unknown flags were provided to open.
 /** @} */
 
 /* Flags for open */
-#define KAS_READ_ALL            (1 << 0)
+#define KAS_READ_ALL                       (1 << 0)
 #define KAS_GET_TAKES_OWNERSHIP            (1 << 1)
+#define KAS_PUT_BORROWS_ARRAY              (1 << 2)
```

Comment:
I don't really see the advantage of making this a global open flag - why not pass a flag to kastore_put(...) each time? It already has the array argument as const, right, and the flags argument is currently unused.